frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pageComponents/PTExemption.js

```
@@ -0,0 +1,149 @@
+  let menu = [];
+
+  const { pathname } = useLocation();
+  const presentInModifyApplication = pathname.includes("modify");
+
+  function usageCategoryMajorMenu(usagecat) {
+    if (userType === "employee") {
+      const catMenu = usagecat
+        ?.filter((e) => e?.code.split(".").length <= 2 && e.code !== "NONRESIDENTIAL")
+        ?.map((item) => {
+          const arr = item?.code.split(".");
+          if (arr.length == 2) return { i18nKey: "PROPERTYTAX_BILLING_SLAB_" + arr[1], code: item?.code };
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)